### PR TITLE
Use of one stream to achieve in-order delivery

### PR DESCRIPTION
**Pull request by mirjak**
One sentences added to address remaining bits for #39

### CODE REVIEW

**Comment by mirjak** [comment]
@britram I didn't replace "five-tuple" with "connection" in this sentence in this PR:
"QUIC’s stream multiplexing feature allows applications to run multiple streams over a single connection, without head-of-line blocking between streams, associated at a point in time with a single five-tuple. "
because it already has connection in the sentence; so we could only remove the last half sentence, but I actually think there is a value in leaving this in because some application may actually care about the 5-tuple.

**Comment by britram** [comment]
WFM. We should probably do an editorial pass to make sure we're using terms like "5-tuple" and "connection" and "path" properly everywhere anyway.

File: draft-ietf-quic-applicability.md [hunk 1]
```diff
@@ -253,20 +253,21 @@ provide an interface to exceptional handling of any stream. There are special
...
-anymore can be used to emulate partialreliability on a message basis. If a QUIC
-receiver has maximum allowed concurrent streams open and the sender on the
-other end indicates that more streams are needed, it doesn't automatically
-lead to an increase of the maximum number of streams by the receiver.
-Therefore it can be valuable to expose maximum number of allowed, currently
-open and currently used streams to the application to make the mapping of
-data to streams dependent on this information.
+Mapping of application data to streams is application-specific and described for
+HTTP/s in {{QUIC-HTTP}}. In general data that can be processed independently,
+and therefore would suffer from head of line blocking, if forced to be received
```
Comment (britram) [inline draft-ietf-quic-applicability.md]:
```suggestion
and therefore would suffer from head of line blocking if forced to be received
```

File: draft-ietf-quic-applicability.md [hunk 2]
```diff
@@ -253,20 +253,21 @@ provide an interface to exceptional handling of any stream. There are special
...
+in order, should be transmitted over different streams. If the application,
```
Comment (britram) [inline draft-ietf-quic-applicability.md]:
```suggestion
in order, should be transmitted over different streams. If the application
```

File: draft-ietf-quic-applicability.md [hunk 3]
```diff
@@ -253,20 +253,21 @@ provide an interface to exceptional handling of any stream. There are special
...
+however, requires certain data to be received in order, the same stream should
```
Comment (britram) [inline draft-ietf-quic-applicability.md]:
```suggestion
requires certain data to be received in order, the same stream should
```

File: draft-ietf-quic-applicability.md [hunk 4]
```diff
@@ -253,20 +253,21 @@ provide an interface to exceptional handling of any stream. There are special
...
+be used for that data. If there is a logical grouping of data chunks or
+messages, streams can be reused, or a new stream can be opened for each
+chunk/message. If one message is mapped on a single stream, resetting the stream
```
Comment (britram) [inline draft-ietf-quic-applicability.md]:
```suggestion
chunk/message. If one message is mapped to a single stream, resetting the stream
```

File: draft-ietf-quic-applicability.md [hunk 5]
```diff
@@ -253,20 +253,21 @@ provide an interface to exceptional handling of any stream. There are special
...
+if the message is not needed anymore, can be used to emulate partial reliability
```
Comment (britram) [inline draft-ietf-quic-applicability.md]:
```suggestion
to expire an unacknowledged message can be used to emulate partial reliability
```

**Pull request merged**